doc/Language/nativecall.pod6

```diff
@@ -653,9 +653,18 @@ say $esponja;
 
 NativeCall also supports native functions that take functions as arguments.  One
 example of this is using function pointers as callbacks in an event-driven
-system.  When binding these functions via NativeCall, one need only provide the
+system.  When binding these functions via NativeCall, one should need only provide the
 equivalent signature as L<a constraint on the code parameter|/type/Signature#Constraining_signatures_of_Callables>:
 
+    # See below--this syntax does not work on Rakudo currently
+    use NativeCall;
+    # void SetCallback(int (*callback)(const char *))
+    my sub SetCallback(&callback:(Str --> int32)) is native('mylib') { * }
+
+However, in the case of NativeCall, currently Rakudo requires a space between the
```

Comment:
Again, this kind of statement is impossible to maintain. Currently when? Up to when? We can leave the issue open if needed, but the documentation should reflect what's happening right now.